scripts/clean_rebuild.sh

```diff
@@ -107,9 +107,9 @@ if [ -n "$ARG_STAGING" ]; then
   CMD_FLAGS="-xs"
 fi
 
-for gitrepo in eosnetworkfoundation/docs \
+for gitrepo in AntelopeIO/docs \
```

Comment:
Seems to me that we would use the tag, that is what GitHub uses for the releases. If you use the branch you might get changes that are as of yet, not in the binaries.

Reply:
Indeed for code changes, but for docs that's usually desirable. We might want to fix inaccuracies or perform cosmetic changes only. Also we don't want to block a release if some docs are pending. Docs just evolve differently than code.

Reply:
Let's discuss in the next All Hands and decide what we want as a team. However, just for the record, that's how we used to do it before - picking up changes in the `release/x.y` branches directly, because any doc additions/changes are of course for that `x.y` release, not a future version.